hash/vsmile_cart.xml

```diff
@@ -165,6 +165,7 @@ Regular game cartridges
 |        | 80-092062(NL)     | Winnie de Poeh - De wonderwereld van Poeh                                                      |
 |   XX   | 80-092062-123(NL) | Winnie de Poeh - De wonderwereld van Poeh (rev. 123, labeled as "My Friends Tigger and Pooh", 52-092062(NL) on back label)
 |        | 80-092063(UK)     | Winnie The Pooh: The Honey Hunt                                                                |
+|   XX   | 80-092063-103(UK) | Winnie The Pooh: The Honey Hunt (52-092063-103(UK) on back label                               |
 |   XX   | 80-092064(GE)     | Pooh - Die Honigjagd                                                                           |
```

Comment:
Are the parentheses in this comment supposed to balance, or is the opening parentheses in “(52-” part of what’s on the back label?